organizations/views.py

```diff
@@ -137,7 +137,7 @@ def send_membership_approved_notification(membership, approved_by):
     subject = _("volunteer-planner.org: Membership approved")
 
     from_email = settings.DEFAULT_FROM_EMAIL
-    reply_to = approved_by.email
+    reply_to = (approved_by.email,)
```

Comment:
"reply-to needs to be a list or tuple" - that one took me by surprise. I wonder why that API would have been changed, but ok... I guess :-)

Reply:
IIRC reply-to is, by RFC, allowed to be a list of "please reply to ..." addresses.
Guess, this API change introduced this change, to be "RFC compliant"?